data/hai/unfettered.txt

```diff
@@ -0,0 +1,161 @@
+
+phrase "Essential Unfettered Mission Fail"
+	word
+		`You have failed an essential Unfettered mission. You may want to reload your previous save if you want to be able to finish the campaign.`
+
+phrase "Essential Unfettered Mission Fail If Killed"
+	word
+		`If you did not kill the duelling ship, you may try this mission again later. Otherwise, you have failed an essential Unfettered mission. You may want to reload your previous save if you want to be able to finish the campaign.`
+
+# using <, > so we can have an empty name
+# could add some depending on the kind of warlord you are (and SB you choose):
+# He-Shi: sneaky, deadly
+# Mem-He: fast, ?
+# To-or: enduring, honest
+substitutions
+	"<uhai>" "Triumph of the True Hai"
+	"<, unfettered title>" ", monkey"
+	"<, unfettered title>" ", space-monkey"
+		has "Unfettered Jump Drive 1: offered"
+	"<, unfettered title>" ", Captain"
+		has "unfettered: beaten the captain"
+	"<, unfettered title>" ", Herald"
+		has "salary: Unfettered"
+	"<, unfettered title>" ", Warlord"
+		"salary: Unfettered" > 1000
+	"<, unfettered title>" ", High Warlord"
+		has "unfettered: high warlord"
+	"<, unfettered title>" ", outcast"
+		has "unfettered: outcast"
+
+
+
+conversation "unfettered outcast"
+	branch "no license"
+		has "license: Unfettered"
+	action
+		# replace it by a wortless outfit
+		clear "license: Unfettered"
+		outfit "Unfettered License"
+	label "no license"
+	action
+		event "unfettered: outcast"
+		event "unfettered: challengers leave"
+	`The local authorities hail you immediately after the destruction of the Alpha ship with a code purple.`
+		to display
+			has "unfettered: destroyed auxiliary"
+	`The local authorities hail you immediately after the destruction of the duelling ship with a code purple.`
+		to display
+			not "unfettered: destroyed auxiliary"
+	branch translator
+		has "language: Hai"
+	`	They speak in Hai but the tone alone is enough to understand what they mean: you have angered them greatly by destroying that ship, and you should leave, or you risk being shot at. You probably will not be offered missions from them in the future.`
+		decline
+	label translator
+	branch "alpha ship"
+		has "unfettered: destroyed auxiliary"
+	`	"You have disappointed us greatly by breaking the sacred rules of the duels! Think very carefully before you make any more mistakes, or it will be war! We have removed any privileges you had, and have added you to the list of shame. Rest assured that no True Hai will offer you special missions any longer, but we'll allow you to ferry cargo. For now."`
+		decline
+	label "alpha ship"
+	`	"How dare you?! This ship was helping us by providing supplies, and you just shoot it down? Disabling it would have been more than fine, looting it tolerated, but destroying the source of our independence? We cannot tolerate it. It matters not who may have drawn the first shot, you have hereby lost all privileges, and it will be war if you make the slightest mistake!"`
```

Comment:
```suggestion
	`	"How dare you?! This ship was helping us by providing supplies, and you just shot it down? Disabling it would have been more than fine, looting it tolerated, but destroying the source of our independence? We cannot tolerate it. It matters not who may have drawn the first shot, you have hereby lost all privileges, and it will be war if you make the slightest mistake!"`
```

Reply:
I think both are correct, I didnt mean it just happened I meant you casually shoot it down